Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayerSkia.cpp

```diff
@@ -135,31 +139,101 @@ class AcceleratedBufferPool {
     RunLoop::Timer m_releaseUnusedBuffersTimer;
 };
 
+class UnacceleratedThreadedRenderingPool {
+    WTF_MAKE_FAST_ALLOCATED();
+public:
+    static UnacceleratedThreadedRenderingPool& singleton()
+    {
+        static NeverDestroyed<UnacceleratedThreadedRenderingPool> s_pool;
+        return s_pool;
```

Comment:
No need to use s_ prefix in this case.

Reply:
Why not? I thought we use s_ for all static vars?

Reply:
Ok, fair enough.